LaderAppen2/app/src/main/res/layout/drive_fragment.xml

```diff
@@ -33,15 +33,16 @@
         app:layout_constraintBottom_toBottomOf="parent"/>
 
     <Button
-        android:id="@+id/btn_drive_auto"
+        android:id="@+id/btnAutoDrive"
```

Comment:
Using camel case, when all other IDs are snake case; why?

Reply:
btn_drive_auto was a previous button that was broken. Using the same ID would keep the issue that was previously there, changing the ID was done to ensure it would not keep the old issue.

Reply:
😡💢💢💢